### PR TITLE
v1.2.1

### PR DESCRIPTION
**Pull request by LukasBures**
- [x] FIX: issue in Batch Split when are more addresses than you can divide into
- [x] Small visual GUI changes 
- [x] Linux Sync issue: Add user defined delay

### CODE REVIEW

File: app/main.js [hunk 1]
```diff
@@ -77,6 +90,7 @@ const defaultSettings = {
     autoLogOffTimeout: 60,
     explorerUrl: "https://explorer.zensystem.io",
```
Comment (cronicc) [inline app/main.js]:
```suggestion
    explorerUrl: "https://explorer.horizen.global",
```

File: app/main.js [hunk 2]
```diff
@@ -77,6 +90,7 @@ const defaultSettings = {
...
     apiUrls: [
+        "https://explorer.horizen.global/insight-api-zen",
```
Comment (cronicc) [inline app/main.js]:
```suggestion
        "https://explorer.horizen.global/api",
```

File: app/main.js [hunk 3]
```diff
@@ -77,6 +90,7 @@ const defaultSettings = {
...
         "https://explorer.zensystem.io/insight-api-zen",
```
Comment (cronicc) [inline app/main.js]:
```suggestion
```

File: app/main.js [hunk 4]
```diff
@@ -85,7 +99,8 @@ const defaultSettings = {
     secureNodePort: 18231,
     domainFronting: false,
     domainFrontingUrl: "https://www.google.com",
-    domainFrontingHost: "zendhide.appspot.com"
+    domainFrontingHost: "zendhide.appspot.com",
```
Comment (cronicc) [inline app/main.js]: I'd prefer removing domain fronting functionality altogether for now as it is not working anymore. (Nonblocking, for future)

File: package.json [hunk 5]
```diff
@@ -35,7 +35,7 @@
     "desktop",
     "electron"
   ],
-  "author": "ZEN team <lukas@zensystem.io>",
+  "author": "Horizen team <lukas@zensystem.io>",
```
Comment (cronicc) [inline package.json]:
```suggestion
  "author": "Horizen team <arizen@horizen.global>",
```

File: .travis.yml [hunk 6]
```diff
@@ -24,10 +25,10 @@ matrix:
     osx_image: xcode9.2
     sudo: required
     language: c
-  - os: windows
-    sudo: required
-    env: TARGET_ARCH=x64
-    language: c
+  # - os: windows
```
Comment (cronicc) [inline .travis.yml]: What is the plan for Windows releases with Travis not working?

File: app/main.js (hunk 4 above)
Comment (gsfakianakis) [reply app/main.js]: Quick removal done.

File: .travis.yml (hunk 6 above)
Comment (gsfakianakis) [reply .travis.yml]: Manual Builds and fix travis ci after the release of v1.2.1

**Pull request closed without merging**